manifests/r/RuneLite/RuneLite/2.2.0/RuneLite.RuneLite.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.0 using YAML parsing
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: RuneLite.RuneLite
+PackageVersion: 2.2.0
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
```

Comment:
```suggestion
InstallerType: inno
Scope: machine
```

Reply:
I most have looked at the wrong PR, it is however user tho